src/QbXml/Objects/IQbListRet.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace QbSync.QbXml.Objects
+{
+    /// <summary>
+    /// Interface for common properties to all QuickBooks List ret objects.
+    /// </summary>
+    public interface IQbListRet : IQbRet
+    {
+        string ListID { get; }
```

Comment:
My other comment above felt alone, so I wanted to add another one to make sure you deliberately set { get; } on your Ret types, but get; set; on everything else?
Reasoning?

Reply:
`IQbRet`, `IQbTxnRet`, and `IQbListRet` only expose the getter because Ret objects are never sent to QB